diff -ru a/deps/googlebenchmark/src/benchmark_register.h b/deps/googlebenchmark/src/benchmark_register.h
--- a/deps/googlebenchmark/src/benchmark_register.h	2019-05-14 03:42:18.000000000 +0800
+++ b/deps/googlebenchmark/src/benchmark_register.h	2021-12-03 04:05:36.282683508 +0800
@@ -1,6 +1,7 @@
 #ifndef BENCHMARK_REGISTER_H
 #define BENCHMARK_REGISTER_H
 
+#include <limits>
 #include <vector>
 
 #include "check.h"
